Vysp3r/ProtonPlus
# :construction: :memo: DO NOT MERGE: Add how to build from source with flatpak

### Category
Documentation

### Overview
Coming soon

### Issue Number
Closes #49 

### New Vars
Created the .gitignore file with three folders.
The meson build output.
```
# Ignore meson build dirs
build
```

The first is the flatpak-builder cache, downloads, etc. The second is flatpak-builder build output.
```
# Ignore flatpak build dirs
.flatpak-builder
build-dir
```

### How Has This Been Tested?
Tested locally on my machine, Debian Bookworm/Testing, and with lxd containers and VM.

- LXD CONTAINER
  - Fedora 37, Debian Bullseye and Arch 
    1. Start the container and configure the dependencies.
    2. Build the application using flatpak-builder.
    3. Expose the host wayland and Xwayland socket inside lxd container.
    4. Launch the app and you can see the app window on the host.

- LXD VIRTUAL MACHINE
  - Debian Bullseye
    1. Start the vm and configure the dependencies.
    2. Build the application using flatpak-builder.
    3. Conect to vm using SPICE (spice-client-gtk).
    4. Launch the app and you can see the app window inside the spice client.

- MY LOCAL MACHINE.
  - Debian Bookworm/Testing
    1. Setup the dependencies.
    2. Build the application using flatpak-builder.
    3. Launch the app and you can see the app window.
